cmd/clusterlink/config/config.go

```diff
@@ -18,10 +18,8 @@ import (
 )
 
 const (
-	// PrivateKeyFileName is the filename used by private key files.
-	PrivateKeyFileName = "key.pem"
-	// CertificateFileName is the filename used by certificate files.
-	CertificateFileName = "cert.pem"
+	// DefaultFabric is the default fabric name.
+	DefaultFabric = "default_fabric"
```

Comment:
I would drop this default. Either use `default` or leave empty and require a fabric name/generate a random one.